repo2docker/app.py

```diff
@@ -560,6 +560,8 @@ def start_container(self):
                     "mode": "rw",
                 }
 
+        # self.environment needs to be modified to match the semantics
+        # of docker's --env option
```

Comment:
Upon further thought, I don't believe this is the correct place to fix the bug. The intention of `start_container` is just that, not start-the-container-after-mucking-with-environment-variables.

I'm investigating fixing the bug during the initial argument processing.